README.md

```diff
@@ -1 +1,84 @@
-# Portfolio
\ No newline at end of file
+# Protfolio
+
+> This is Portfolio project in Mobile View only with two sections.This project developement assing by Microverse.
+> In the Mobile View Portfolio there are few important sections are visible.
+> This project has done after getting templete from FIGMA templete #4.
+
+![Portfolio in Mobile View with two sections](2sectionmobileview.png)
+
+## Built With
+
+- HTML
+- CSS > FlexBox
+
+## Extensions / Plugins
+
+Linters Installed
+-Lighthouse
+-Webhint
+-Stylelint
+-ESLint
+-node_modules checker
+
+
+
+## Getting Started
+- Open GitHub URL of repositry.
+- Download Code repo in zip.
+- Extract and open repo folder into visual studio code or any other Text Editor Software.
+- GitHub Repositry URL: https://github.com/MudasirSherwani/Portfolio
+  To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Visual Studio
+- Knowldege About HTML / CSS
+Linters
+- Lighthouse
+- Webhint
+- Stylelint
+- ESLint
+- node_modules checker
+- Github
+
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
```

Comment:
- [ ] Kindly remove unused headers in your README, so it looks more professional